dracut/modules.d/90kiwi-live/kiwi-live-lib.sh

```diff
@@ -1,6 +1,17 @@
 #!/bin/bash
 type getarg >/dev/null 2>&1 || . /lib/dracut-lib.sh
 
+function getOverlayBaseDirectory {
+    # initialize and print overlay base directory below which
+    # the iso and overlayfs based mountpoints are managed
+    local overlay_base=/overlay
+    mkdir -p "${overlay_base}"
+    if ! mountpoint -q "${overlay_base}";then
+        mount -t tmpfs tmpfs "${overlay_base}"
```

Comment:
I am wondering if it would make sense to customize the tmpfs constraints for the non persistent overlayfs from the kernel command line, in fact, it means only adding the `-o size=<custom_value>` here. The default value is  `size=50%`. 

Probably something like `rd.kiwi.overlay.size=`. Also it could eventually be meaningful for persistent overlayfs. Anyways this is an enhancement not strictly related to the issue being solved in this PR.

Reply:
very good point. Yes let's add that

Reply:
done. I used rd.live.overlay.size as this matches the other available options. For the overlayroot options I chose rd.root.overlay.size

I also updated the docs